File: Práctica1/Sesion1/Sesion1_Java/src/main/java/Sesion1/CarreraMontana.java
package Sesion1;

public class CarreraMontana extends Carrera {
    
    public CarreraMontana() {
        this.tasaAbandono = 20;
    }
    
    @Override
    public String toString() {
        return "CarreraMontana{" + super.toString() + "}";
    }
}

File: Practica1/Sesion1/Sesion1_Ruby/Ruby_Prototypes/CarreraMontana.rb
require('./Carrera.rb');
require('./BicicletaMontana.rb');

module Ruby_Prototypes
	public
	class CarreraMontana < Carrera
		public
    def initialize()
      super
      @biciPrototype = BicicletaMontana.createBici
      @abandono = 0.2
    end
    
    def to_s
      "Carrera de montana con #{@bicicletas.size} bicicletas"
    end
	end
end
